tests/api/test_overdrive.py

```diff
@@ -1566,6 +1567,91 @@ def test_cannot_fulfill_error_audiobook(self):
         with pytest.raises(CannotFulfill):
             od_api.fulfillment_authorization_header
 
+    def test_no_drm_fulfillment(self):
+        patron = self._patron()
+        work = self._work(with_license_pool=True)
+        patron.authorization_identifier = "barcode"
+        self._default_collection.external_integration.protocol = "Overdrive"
+        self._default_collection.external_account_id = 1
+        self._default_collection.external_integration.setting(
+            OverdriveConfiguration.OVERDRIVE_CLIENT_KEY
+        ).value = "user"
+        self._default_collection.external_integration.setting(
+            OverdriveConfiguration.OVERDRIVE_CLIENT_SECRET
+        ).value = "password"
+        self._default_collection.external_integration.setting(
+            OverdriveConfiguration.OVERDRIVE_WEBSITE_ID
+        ).value = "100"
+
+        od_api = OverdriveAPI(self._db, self._default_collection)
+        od_api._server_nickname = OverdriveConfiguration.TESTING_SERVERS
+
+        # Mock out the flow
+        od_api.get_loan = MagicMock(
+            return_value={
+                "reserveId": "A4466636-34F5-495A-92EE-3A9C701F46CF",
+                "crossRefId": 432890,
+                "expires": "2022-07-27T10:38:06Z",
+                "isFormatLockedIn": False,
+                "formats": [
+                    {
+                        "reserveId": "A4466636-34F5-495A-92EE-3A9C701F46CF",
+                        "formatType": "ebook-overdrive",
+                        "links": {
+                            "self": {
+                                "href": "https://example.org/v1/patrons/me/checkouts/A4466636-34F5-495A-92EE-3A9C701F46CF/formats/ebook-overdrive",
+                                "type": "application/vnd.overdrive.circulation.api+json",
+                            }
+                        },
+                        "linkTemplates": {
+                            "downloadLink": {
+                                "href": "https://example.org/v1/patrons/me/checkouts/A4466636-34F5-495A-92EE-3A9C701F46CF/formats/ebook-overdrive/downloadlink?errorpageurl={errorpageurl}&odreadauthurl={odreadauthurl}",
+                                "type": "application/vnd.overdrive.circulation.api+json",
+                            },
+                            "downloadLinkV2": {
+                                "href": "https://example.org/v1/patrons/me/checkouts/A4466636-34F5-495A-92EE-3A9C701F46CF/formats/ebook-overdrive/downloadlink?errorurl={errorurl}&successurl={successurl}",
+                                "type": "application/vnd.overdrive.circulation.api+json",
+                            },
+                        },
+                    }
+                ],
+                "checkoutDate": "2022-07-26T10:38:06Z",
+            }
+        )
```

Comment:
Might be worthwhile to pull this from a file in `tests/api/files/overdrive` and pull it in from there.